crates/typst-library/src/html/dom.rs

```diff
@@ -470,6 +483,51 @@ pub mod tag {
         wbr
     }
 
+    /// Can the content of a node with the tag be surrounded by whitespace without impacting the document?
+    ///
+    /// This is an underapproximation.
+    pub fn is_block_by_default(tag: HtmlTag) -> bool {
```

Comment:
Please add some explanation that this is based on `display: block` and that it can be overridden by CSS and also add a link to the relevant source(s).

Reply:
I did this in [e156378](https://github.com/typst/typst/pull/5533/commits/e15637813e3da0f9331aac9d1b113636c15b0215).